pdfminer/pdffont.py

```diff
@@ -509,12 +509,28 @@ def __init__(self, descriptor, widths, default_width=None):
         self.glyph_bounding_box = None
         self.font_bounding_box = None
 
+        # Ascent: The maximum height above the baseline reached by glyphs in this font,
+        # excluding the height of glyphs for accented characters.
+        # Descent: The maximum depth below the baseline reached by glyphs in this font. The value
+        # is a negative number.
+        # Units are in 'glyph space'
+        self.default_ascent = 100
+        self.default_descent = -1000
```

Comment:
Is this right? Highly unintuitive that descent is 10x ascent

Reply:
I agree. But it produces text boxes that look right. I think that in most PDFs ascent and descent are 1000, -100 respectively. This PDF is unique in some ways =(

Reply:
There are strong limitations to setting defaults I guess.